<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerConfigDialogForm</class>
 <widget class="QDialog" name="SeerConfigDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seer - Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="contentsListWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>25</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="pagesStackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>75</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="SeerGdbConfigPage" name="gdbConfigPage" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="SeerEditorConfigPage" name="editorConfigPage" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SeerGdbConfigPage</class>
   <extends>QWidget</extends>
   <header location="global">SeerGdbConfigPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SeerEditorConfigPage</class>
   <extends>QWidget</extends>
   <header location="global">SeerEditorConfigPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SeerConfigDialogForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SeerConfigDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
